common/vts.proto

```diff
@@ -15,6 +19,122 @@ message CloseArgs {
 }
 
 message CloseResponse {
+    Status status = 1;
+}
+
+message OpenArgs {
+}
+
+message OpenResponse {
+    Status status = 1;
+}
+
+enum TaType {
```

Comment:
Should be `TAType` to be consistent with golang naming conventions?

Reply:
SGTM